classes/class_collisionobject2d.rst

```diff
@@ -169,7 +169,7 @@ Property Descriptions
 
 The physics layers this CollisionObject2D is in. Collision objects can exist in one or more of 32 different layers. See also :ref:`collision_mask<class_CollisionObject2D_property_collision_mask>`.
 
-**Note:** Object A can detect a contact with object B only if object B is in any of the layers that object A scans. See `Collision layers and masks <https://docs.godotengine.org/en/latest/tutorials/physics/physics_introduction.html#collision-layers-and-masks>`__ in the documentation for more information.
+**Note:** Object A can detect a contact with object B only if object B is in any of the layers that object A scans. See `#collision-layers-and-masks <../tutorials/physics/physics_introduction.html#collision-layers-and-masks>`_ in :doc:`../tutorials/physics/physics_introduction` in the documentation for more information.
```

Comment:
This doesn't seem to be working properly, it should have used "Collision layers and masks" as title.

Reply:
Fixed.